server/routes/config/proposedAddress.js

```diff
@@ -45,6 +45,18 @@ module.exports = {
         licenceSection: 'curfewAddress',
         nextPath: '/hdc/proposedAddress/submit/'
     },
+
+    curfewAddressConfirmation: {
+        licenceSection: 'curfewAddress',
+        nextPath: null,
+        fields: [
+            {landLordHDCConsent: {}},
+            {hasElectricitySupply: {dependantOn: 'landLordHDCConsent', predicate: 'Yes'}},
+            {homeVisitConducted: {dependantOn: 'landLordHDCConsent', predicate: 'Yes'}},
+            {managedSafely: {}},
+            {managedSafelyReasons: {dependantOn: 'managedSafely', predicate: 'No'}}
```

Comment:
Need to change all these to dependent 'e' not 'a'

Reply:
Can't spell to save a kitten 😅